### DISCUSSION TITLE
Im using android phone, can't afford laptop. My issue is in this picture

### DISCUSSION THREAD

**Discussion by Alfred1023** [revoxhere/duino-coin#1326] [Q&A]
![Screenshot_20220131-012632](https://user-images.githubusercontent.com/98298359/151710221-00747a4a-9e09-42cf-8437-b38d5faf3463.jpg)

**Comment by LDarki** [comment]
Hi, you need the esp8266 board driver.

Go to your settings of the arduino IDE.
Enter a URL so that the IDE gets access to the drivers. Enter the following URL: ```http://arduino.esp8266.com/stable/package_esp8266com_index.json```

Now open the Board Manager of the Arduino IDE and search for "esp".
There you get a result called "esp2866". Download the pack.
You now get new Boards to select.

For example select the NodeMCU 1.0(ESP-12E Module).

But idk how to do it in this app.

**Reply by Alfred1023** [reply]
Im sorry im using phone only, not computer

**Reply by LDarki** [reply]
You have the ESP8266 Board selected in the settings of the app?

**Comment by Alfred1023** [comment]
yes i have

**Comment by Alfred1023** [comment]
Can you program? and i download the file?

**Reply by LDarki** [reply]
Well, i can`t do anything about that but you can try replacing that function:
[Download](https://anonfiles.com/f8c0G2E4xc/esp8266_code_ino)

Edit: i can`t upload the .ino file here so, i use anonfiles
